package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "asset-depreciation-calculator",
-  "version": "0.0.5",
+  "version": "0.0.6",
```

Comment:
💭 I think we could stay in 0.0.1 since this is going to be the first release 🤔

Reply:
I'll bump it to `1.0.0` once it's merged to master.
the current version is published under `beta` tag, so it's technically not released yet.